File: server/plugin/pcdn/router/profit.go
package router

import "github.com/gin-gonic/gin"

// ProfitRouter 利润大盘路由（admin 组）
type ProfitRouter struct{}

// InitProfitRouter 挂载利润路由
func (r *ProfitRouter) InitProfitRouter(group *gin.RouterGroup) {
	group.GET("profit/summary", profitApi.GetProfitSummary)
	group.GET("profit/revenueByPlatform", profitApi.GetRevenueByPlatform)
	group.GET("profit/costByOwner", profitApi.GetCostByOwner)
	group.GET("profit/trend", profitApi.GetProfitTrend)
}

File: web/src/plugin/pcdn/api/profit.js
import service from '@/utils/request'

export const getProfitSummary = (params) => service({ url: '/pcdn/admin/profit/summary', method: 'get', params })
export const getRevenueByPlatform = (params) => service({ url: '/pcdn/admin/profit/revenueByPlatform', method: 'get', params })
export const getCostByOwner = (params) => service({ url: '/pcdn/admin/profit/costByOwner', method: 'get', params })
export const getProfitTrend = (params) => service({ url: '/pcdn/admin/profit/trend', method: 'get', params })
